lib/shape.ml

```diff
@@ -98,6 +98,8 @@ let rotate degrees shape =
   | Ellipse ellipse -> Ellipse { c = (rot ellipse.c degrees); rx = ellipse.rx; ry = ellipse.ry }
   | Line _line -> failwith "Not Implemented"
 
+let comp f g x = g (f x)
```

Comment:
It might be better to call this `compose` for clarity.

Reply:
Done!